src/test/kotlin/study/PersonTest.kt

```diff
@@ -0,0 +1,33 @@
+package study
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class PersonTest {
+    @Test
+    fun `이름 붙인 인자`() {
+        val person = Person(name = "권중훈", age = 27, nickname = "gjunghoon")
+        assertThat(person.name).isEqualTo("권중훈")
+        assertThat(person.age).isEqualTo(27)
+        assertThat(person.nickname).isEqualTo("gjunghoon")
+    }
+
+    @Test
+    fun `널 타입`() {
+        val person = Person("권중훈", 20, null)
+        assertThat(person.nickname).isNull()
+    }
+
+    @Test
+    fun `기본 인자`() {
+        val person = Person("권중훈", 20)
+        assertThat(person.name).isEqualTo(person.nickname)
+    }
```

Comment:
```suggestion
    @Test
    fun `기본 인자`() {
        val person = Person("권중훈", 20)
        assertThat(person.name).isEqualTo("권중훈")
    }
```

`person.nickname` 도 좋지만, 저는 상수로 작성된 코드가 더 잘 읽히더라구요!
취향의 영역이므로 넘어가주셔도 됩니다. 😃